client/clientBean.go

```diff
@@ -65,9 +66,10 @@ type (
 		SetRemoteFrontendClient(cluster string, client frontend.Client)
 	}
 
-	// DispatcherProvider provides a diapatcher to a given address
+	// DispatcherProvider provides a dispatcher to a given address
 	DispatcherProvider interface {
 		Get(name string, address string) (*yarpc.Dispatcher, error)
```

Comment:
Can you rename it to GetTchannel to make it explicit?

Reply:
Yes, updated.